lib/fx/adapters/postgres.rb

```diff
@@ -125,7 +125,11 @@ def update_trigger(name, on:, sql_definition:)
       #
       # @return [void]
       def drop_function(name)
-        execute "DROP FUNCTION #{name}();"
+        fns = functions.select { |function| function.name == name.to_s }
```

Comment:
I generally try to shy away from using acronyms as variable names. Keeping a local variable for a single loop seems redundant. So what do you think of extracting this into a private method along the lines of `functions_with_matching_signature(name)` and use that directly in the `.each` loop?